gormsupport/postgres.go

```diff
@@ -7,6 +7,7 @@ const (
 	errUniqueViolation     = "23505"
 	errForeignKeyViolation = "23503"
 	errInvalidCatalogName  = "3D000"
+	errDataException       = "22" // This class contains many data representation related errors. See https://www.postgresql.org/docs/9.5/static/errcodes-appendix.html
```

Comment:
Shall we use non-version link: https://www.postgresql.org/docs/current/static/errcodes-appendix.html

Reply:
Yes, I have changed it https://github.com/fabric8-services/fabric8-wit/pull/2321/commits/a15afc995a10123d13ac17a3364eb8bac7e73ade